release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -1971,8 +1964,10 @@ properly.
 
 Issued: 2020-07-13
 
-{product-title} release 4.5 is now available. The list of container images and
-bug fixes includes in the update are documented in the
+Red Hat did not publicly release {product-title} 4.5.0 or 4.5.1 as the GA
```

Comment:
But we did ship 4.5.1, so I don't think this is accurate.

Reply:
@sferich888 Could we instead just state that it's not available for upgrades? And leave the rest as-is?:
```
Upgrades to version 4.5.1 are blocked; the 4.5.1 advisories are available for reference only.
```